docs/application/dotnet/guides/connectivity/stc.md

```diff
@@ -0,0 +1,167 @@
+# Smart Traffic Control
+
+
+Smart Traffic Control (STC) provides extensible packet-level control services, including per-app data usage, total data quota, and background app's data saving.
+
+The main features of the Tizen.Network.Stc namespace include:
+
+- Retrieve Data Usage for System
+
+  You can [retrieve the network data consumed by the system](#retrieve-data-usage-for-system).
+
+- Retrieve Data Usage for Applications
+
+  You can [retrieve the network data consumed by the applications](#retrieve-data-usage-for-applications).
+
+
+> **Note**  
+
```

Comment:
@prasadam-p > is needed at the additional line, also.

\> \*\*Note**
\>
\> You can test the STC functionality on a target device only. The emulator does not support this feature.

For now the note is broken. Please add > at the line 18.

![image](https://user-images.githubusercontent.com/9958374/70132868-614e1980-16c8-11ea-9b47-9b24c204fd0b.png)

Reply:
done

Reply:
@prasadam-p I can't see the update commented here. Would you please check again the git push?

Reply:
Done, Please check